functions/base.php

```diff
@@ -828,7 +828,11 @@ function get_search_results(
 	if (strlen($query) > 0){
 		$query_string = http_build_query($arguments);
 		$url          = $search_url.'?'.$query_string;
+<<<<<<< HEAD
```

Comment:
Unresolved merge conflict needs to be fixed.

Reply:
Fixed.